src/widgets/_viewswitcher.scss

```diff
@@ -0,0 +1,92 @@
+viewswitcher button {
+    margin: 0;
+    padding: 0;
+    border-radius: 0;
+    border-top: 0;
+    border-bottom: 0;
+    box-shadow: none;
+    font-size: rem(13px);
```

Comment:
I am once again asking why you are setting a different font size here at all?

Reply:
Try it in the demo, make one of the viewswitcher titles a really long one, and test with and without.
That should explain why.

Reply:
After some testing, I think I'll drop this, might be fine without this.